src/test/scala/temple/builder/ServerBuilderTest.scala

```diff
@@ -108,7 +108,10 @@ class ServerBuilderTest extends FlatSpec with Matchers {
         StructRoot(
           name = "Test",
           opQueries = SortedMap(
-            // TODO: Coming soon
+            Create -> "INSERT INTO test (favourite_colour, bed_time, favourite_number) VALUES ($1, $2, $3) RETURNING favourite_colour, bed_time, favourite_number;",
+            Read   -> "SELECT favourite_colour, bed_time, favourite_number FROM test WHERE id = $1;",
+            Update -> "UPDATE test SET favourite_colour = $1, bed_time = $2, favourite_number = $3 WHERE id = $4 RETURNING favourite_colour, bed_time, favourite_number;",
+            Delete -> "DELETE FROM test WHERE id = $1;",
```

Comment:
I don't think we've included trailing semi-colons elsewhere

Reply:
@smithwjv this is from mkCode.stmt, see the existing queries in https://github.com/TempleEight/temple/blob/develop/src/test/scala/temple/builder/ServerBuilderTest.scala